WidgetExtension/Up Next/UpNextMediumWidgetView.swift

```diff
@@ -71,13 +71,11 @@ struct MediumFilterView: View {
         GeometryReader { geometry in
             VStack(alignment: .leading, spacing: 0) {
                 HStack(alignment: .top) {
-                    if let filterName = filterName {
-                        Text(filterName)
-                            .font(.callout)
-                            .fontWeight(.regular)
-                            .foregroundColor(Color.secondary)
-                            .frame(height: 18)
-                    }
+                    Text(filterName)
+                        .font(.callout)
+                        .fontWeight(.regular)
+                        .foregroundColor(Color.secondary)
+                        .frame(height: 18)
```

Comment:
Similar to the view model above, `filterName` is not `Optional` so we don't need the optional binding.